### PR TITLE
Utilizing manifest listed payload

### PR DESCRIPTION
**Pull request by deepsm007**
Adding aws-ovn-multi job utilizing heterogeneous payload to deploy homogeneous amd64 cluster

/cc @stbenjam  @dgoodwin

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml [hunk 1]
```diff
@@ -456,6 +469,15 @@ tests:
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
     workflow: openshift-e2e-aws-sdn
+- as: e2e-aws-ovn-multi
```
Comment (dgoodwin) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:472]: Sippy's expecting multiRegex = regexp.MustCompile(`(?i)-heterogeneous`)

Can we rename so it picks up the right variant there?

Or is this a multi-arch payload, but regular amd64 job?

Comment (deepsm007) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:472]: It is still an amd64 job with multiarch payload

Comment (deepsm007) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:472]: multiRegex are used for true heterogeneous deployments only (cp=amd64, nodes=amd64+ arm64)

**Pull request merged**